docs/networks/l16-testnet.md

```diff
@@ -97,95 +118,68 @@ make stop
 make reset
 ```
 
-### Check the network status
+### Check the Network Status
 
-You can see you node in the following page:
+You can see your node on the following page:
 
 1. [https://ethstats.l16.d.lukso.dev/](https://ethstats.l16.d.lukso.dev/)
 2. [https://consensus.stats.beta.l16.lukso.network](https://consensus.stats.beta.l16.lukso.network)
 
-## How to become a validator
-
-### Request some test tokens: LYXt
-
-Make sure to install [MetaMask](https://metamask.io/).
-
-To add the L16 Beta Network to [MetaMask](https://metamask.io/) you can follow this tutorial: [https://blog.suhailkakar.com/add-custom-networks-to-metamask](https://blog.suhailkakar.com/add-custom-networks-to-metamask)
-
-**Use the following parameters:**
+## How to become a Validator
 
-| Setting            | Value                                                                                            |
-| ------------------ | ------------------------------------------------------------------------------------------------ |
-| Network Name       | L16 Beta                                                                                         |
-| New RPC URL        | [https://rpc.beta.l16.lukso.network](https://rpc.beta.l16.lukso.network)                         |
-| Chain ID           | 19051978 (0x122B5CA)                                                                             |
-| Currency Symbol    | LYXt                                                                                             |
-| Block Explorer URL | [https://consensus.stats.beta.l16.lukso.network](https://consensus.stats.beta.l16.lukso.network) |
+### Request some LYXt Tokens:
 
-Then select `L16 Beta` from the dropdown
+1. Make sure to install [MetaMask](https://metamask.io/).
 
-![metamask_dropdown.png](./static/l16/metamask_dropdown.png)
+2. Add the L16 Beta Testnet to [MetaMask](https://metamask.io/), by following [this tutorial](https://blog.suhailkakar.com/add-custom-networks-to-metamask).
+3. Use the MetaMask parameters from above and select `L16 Beta` from the dropdown.
+4. Copy your MetaMask address to the clipboard using the box icon.
+5. Visit the [L16 Faucet](https://faucet.beta.l16.lukso.network), paste the copied address and request your test tokens.
+6. Wait for the transaction to go through and check the balance in your MetaMask. You should have received 35 LYX.
 
-Copy address to the clipboard using the box icon
+### Create a Wallet and deposit Data
```

Comment:
I'm not sure i get the "when uppercase" rule.

why `W` and `D` for data but `d` for deposit?

I prefer to only capitalise the first word, at least the rule is easier to follow.

Reply:
Yes, you're right. This time uppercase was not correct. I also applied some changes to other headings that were still missing. 

Small Explanation of rules:
I usually look at the side menu and check if everything has a unified styling and length (always use the 2-row table of contents or stick to the single row one, so navigation has uniform spacing).

We should stick to using uppercase when headings are vital points or single words- such as:
- Create a Wallet,
- Deposit the Data,
- MetaMask.

We should leave out "and, or, plus, etc." because these words imply skipping steps. users should feel comfortable reading through the table of contents on the right side as a "todo-list with one-by-one tasks."

On more minor headings that do not show up on the side menu or belong to a FAQ, we still use sentences as headings, include the dot at the end, and keep them impersonal. Personal references should be outsourced to descriptions and texts. At least that's my input from completed guiding/writing courses.